.env

```diff
@@ -1,22 +1,23 @@
 RUST_TOOLCHAIN=nightly-2022-07-24
 POLKADOT_BUILD_BRANCH=release-v0.9.30
 
-POLKADOT_MAINNET_BRANCH=release-v0.9.29
-STATEMINT_BUILD_BRANCH=release-parachains-v9271
-ACALA_BUILD_BRANCH=2.9.6
-MOONBEAM_BUILD_BRANCH=v0.26.1
-UNIQUE_MAINNET_TAG=v924010-old-tests-fixes
+POLKADOT_MAINNET_BRANCH=release-v0.9.30
+STATEMINT_BUILD_BRANCH=release-parachains-v9320
+ACALA_BUILD_BRANCH=2.10.1
+MOONBEAM_BUILD_BRANCH=runtime-1901
+UNIQUE_MAINNET_BRANCH=v930013
```

Comment:
wrong @mrshiposha @BuddyGlas

Reply:
Will set `UNIQUE_MAINNET_BRANCH=v930033`

Reply:
there is no v930013 release